grafana-plugin/src/components/Text/Text.tsx

```diff
@@ -81,8 +83,10 @@ const Text: TextInterface = (props) => {
     setValue(e.target.value);
   }, []);
 
+  const CustomTag = customTag || (`span` as any);
```

Comment:
We should get rid of `as any` by changing the type of `customTag` prop from `string` into union type (like ``span' | 'div' | 'h1' | 'h2'` etc). Then you can also set default prop when destructuring `customTag = 'span', ...rest } = props;`

Reply:
Right, just that every typing I tried it was complaining it's not on the IDE's taste :laughing: